ftp/src/test/java/org/apache/pekko/stream/connectors/ftp/FtpsWithTrustAndKeyManagersStageTest.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
New files should use the standard ASF header. Only code written by Lightbend gets the header you've added here.

https://github.com/apache/incubator-pekko-connectors/blob/main/project/MetaInfLicenseNoticeCopy.scala#L1-L16

Reply:
Fixed